client/app/veeam-enterprise/veeam-enterprise.controller.js

```diff
@@ -0,0 +1,13 @@
+class VeeamEnterpriseCtrl {
+  constructor($scope, $stateParams, VeeamEnterpriseService) {
+    this.$scope = $scope;
```

Comment:
I think you can safely remove `$scope`, Isn't it?

Reply:
Yes :)
Done